geniza/corpus/tests/test_corpus_models.py

```diff
@@ -403,15 +403,22 @@ def test_index_data(self, document):
         assert index_data["pgpid_i"] == document.pk
         assert index_data["type_s"] == str(document.doctype)
         assert index_data["description_t"] == document.description
-        assert index_data["notes_t"] == document.notes
-        assert index_data["needs_review_t"] == document.needs_review
+        assert index_data["notes_t"] is None  # no notes
+        assert index_data["needs_review_t"] is None  # no review notes
         for frag in document.fragments.all():
             assert frag.shelfmark in index_data["shelfmark_ss"]
         for tag in document.tags.all():
             assert tag.name in index_data["tags_ss"]
         assert index_data["status_s"] == "Public"
         assert not index_data["old_pgpids_is"]
 
+        # test with notes and review notes
+        document.notes = "FGP stub"
+        document.needs_review = "check description"
+        index_data = document.index_data()
+        assert index_data["notes_t"] == document.notes
+        assert index_data["needs_review_t"] == document.needs_review
```

Comment:
why do we index the "needs review" notes?

Reply:
oh, is it for #114? makes sense, if so

Reply:
yes, exactly — we need admin fields and suppressed documents indexed in solr for the admin document search, and will have to make sure we exclude them from the public search